Fix destroot by gzipping manpage before trying to install it.
http://notmuchmail.org/pipermail/notmuch/2015/020103.html
--- doc/Makefile.local.orig	2014-11-14 13:35:40.000000000 -0600
+++ doc/Makefile.local	2015-04-29 21:35:59.000000000 -0500
@@ -63,7 +63,7 @@
 ifeq ($(HAVE_DOXYGEN),1)
 MAN_GZIP_FILES += ${APIMAN}.gz
 apidocs: $(APIMAN)
-install-apidocs: apidocs
+install-apidocs: ${APIMAN}.gz
 	mkdir -p "$(DESTDIR)$(mandir)/man3"
 	install -m0644  $(DOCBUILDDIR)/man/man3/*.3.gz  $(DESTDIR)/$(mandir)/man3
 
